static/app/views/starfish/components/pageFilterContainer.tsx

```diff
@@ -29,6 +30,10 @@ function StarfishPageFilterContainer(props: {children: React.ReactNode}) {
     });
   }
 
+  if (invalidDateFilters) {
+    return null;
+  }
```

Comment:
Could this ever result in a the screen being blank for a long period of time?

Reply:
It should always be almost instant because it only depends on the `router.replace`.

Reply:
Makes sense! Just double checking